src/content/blog/2024-02-21-eslint-2023-year-review.md

```diff
@@ -29,65 +29,65 @@ Just before the end of the year, we [released ESLint v9.0.0-alpha.0](https://esl
 
 The team spends much of the year anticipating new ECMAScript features and updating ESLint accordingly. 2023 was an outlier in that most of the changes weren't syntax-related, and so there wasn't a lot of additional work. The most significant syntax change, [hashbang comments](https://github.com/tc39/proposal-hashbang), is something that ESLint already handled to support Node.js.
 
-## Income review
+## Financials review
 
-During 2023, ESLint received $179,601.53 USD from all of our income sources. You can see the breakdown of our income sources in the following table.
+The next few sections dig into the project financials. All of this information is gathered from the Open Collective transaction list with the exception of Tidelift payments (which are retrieved from Tidelift directly). Because transactions come in all throughout the course of a day, we decided to consider transactions beginning on January 1, 2023 at midnight UTC and ending on December 31, 2023 at 11:59 UTC as our fiscal 2023.
+
+### Income review
+
+During 2023, ESLint received $183,066.53 USD from all of our income sources. You can see the breakdown of our income sources in the following table.
 
 | **Source** | **Amount** |
 |-----------|-------------|
-| Open Collective | $117,336.30 |
-| Tidelift | $30,077.84 |
+| Open Collective | $120,801.30 |
 | GitHub Sponsors | $25,190.89 |
+| Tidelift | $30,077.84 |
 | Carbon Ads | $6,380.89 |
 | Thanks.dev | $524.61 |
 | Stackaid.us | $91.00 |
-| **Total** | **$179,601.53** |
+| **Total** | **$183,066.53** |
 
-As usual, most of our donations came through Open Collective. Tidelift (slight up from 2022) surpassed GitHub Sponsors donations (slightly down from 2022) in 2023. Notably, our website ad revenue dropped roughly 50% from 2022, following an overall softening of the display ad business. We'll be looking at options to improve our website ad revenue in 2024.
+As usual, most of our donations came through Open Collective. Notably, our website ad revenue dropped roughly 50% from 2022, following an overall softening of the display ad business. We'll be looking at options to improve our website ad revenue in 2024.
```

Comment:
Was this because of https://github.com/eslint/eslint.org/issues/520? Also, how do we track website traffic? Are we using Google analytics?

Reply:
I explained:

> following an overall softening of the display ad business

This wasn't specific to us. I discussed with the Carbon Ads folks and they told me they saw this same drop across the board.